en/docs/deploy-and-publish/deploy-on-gateway/choreo-connect/production-deployment-guideline.md

```diff
@@ -7,6 +7,9 @@ You can deploy Choreo Connect using one of the following options.
 - [API Manager as Control Plane]({{base_path}}/deploy-and-publish/deploy-on-gateway/choreo-connect/concepts/apim-as-control-plane/)
 - [Standalone Gateway]({{base_path}}/deploy-and-publish/deploy-on-gateway/choreo-connect/concepts/as-a-standalone-gateway/)
 
+!!! note
+    Choreo Connect Docker based deployment option provided **only for the tryout purpose.** Docker based deployment option is **not recommended to use in production deployments**. For production deployments you can use Kubernetes based Choreo Connect deployment with [Helm artifacts]({{base_path}}/deploy-and-publish/deploy-on-gateway/choreo-connect/getting-started/deploy/cc-on-kubernetes-with-apim-as-control-plane-helm-artifacts/) or [YAML artifacts]({{base_path}}/deploy-and-publish/deploy-on-gateway/choreo-connect/getting-started/deploy/cc-on-kubernetes-with-apim-as-control-plane/).
```

Comment:
Kubernetes should be used for Choreo Connect production deployments. The common guidelines for deploying Choreo Connect in a **production environment** using the [Helm deployment option](base path/deploy-and-publish/deploy-on-gateway/choreo-connect/getting-started/deploy/cc-deploy-overview/) are provided below.